documentation/docs/20-commands/40-sv-migrate.md

```diff
@@ -18,6 +18,10 @@ npx sv migrate [migration]
 
 Upgrades a Svelte 4 app to use Svelte 5, and updates individual components to use [runes](../svelte/what-are-runes) and other Svelte 5 syntax ([see migration guide](../svelte/v5-migration-guide)).
 
+### `self-closing-tags`
+
+Replaces all the self-closing non-void elements in your `.svelte` files. See the [pull request](https://github.com/sveltejs/kit/pull/12128) for more details.
```

Comment:
There is probably a better wording for the first part, this was stolen from the linked pr.